### PR TITLE
Add created by label

### PR DESCRIPTION
**Pull request by QuentinBisson**
Add the created-by label to know what component created which resource. This is especially useful with operators or for auditing

### CODE REVIEW

**Comment by QuentinBisson** [comment]
image

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-27612--kubernetes-io-master-staging.netlify.app/docs/concepts/overview/working-with-objects/common-labels/

File: content/en/docs/concepts/overview/working-with-objects/common-labels.md [hunk 1]
```diff
@@ -54,6 +55,7 @@ metadata:
     app.kubernetes.io/component: database
     app.kubernetes.io/part-of: wordpress
     app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/created-by: helm
```
Comment (kbhawkey) [inline content/en/docs/concepts/overview/working-with-objects/common-labels.md:58]: Is `helm` the correct value?

Comment (QuentinBisson) [inline content/en/docs/concepts/overview/working-with-objects/common-labels.md:58]:
```suggestion
    app.kubernetes.io/created-by: controller-manager
```

Comment (QuentinBisson) [reply content/en/docs/concepts/overview/working-with-objects/common-labels.md]: I forgot to change it to the example above, thank you :)

**Review by onlydole — COMMENTED** [review]
Thank you @QuentinBisson for this change and @kbhawkey for the review!

This looks good to me!

/lgtm

**Review by irvifa — COMMENTED** [review]
/approve

**Pull request merged**